setup.cfg

```diff
@@ -32,6 +32,7 @@ install_requires =
     invenio-administration>=2.0.0,<3.0.0
     lxml>=4.5.0
     PyYAML>=5.4.1
+    s3fs>=2024.6.1
```

Comment:
this should be installed as an extra, otherwise everyone who installs invenio-vocabularies gets also s3fs, which is not desired by default.
In order to make it an extra you will need to make sure the module works also without this package installed. Look at the oaipmh below for inspiration